components/gitpod-db/go/oidc_client_config.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License.AGPL.txt in the project root for license information.
+
+package db
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/google/uuid"
+	"gorm.io/datatypes"
+	"gorm.io/gorm"
+	"time"
+)
+
+type OIDCClientConfig struct {
+	ID uuid.UUID `gorm:"primary_key;column:id;type:char;size:36;" json:"id"`
+
+	Issuer string `gorm:"column:issuer;type:char;size:255;" json:"issuer"`
```

Comment:
what is the issuer referring to? a user?

Reply:
It's the issuer as per the OIDC spec. It can be google for @google.com domains, but other one also.

Reply:
The Issuer (URL) is the ID of a IdP. That's also what is used to associate and validate the ID-tokens.

Reply:
Is it Issuer or Issuer Identifier?
From the [spec](https://openid.net/specs/openid-connect-core-1_0.html)

> Issuer
>    Entity that issues a set of Claims.
> 
> Issuer Identifier
>    Verifiable Identifier for an Issuer. An Issuer Identifier is a case sensitive URL using the https scheme that contains scheme, host, and optionally, port number and path components and no query or fragment components.

Reply:
It is the `Issuer Identifier` indeed. But it's shortened to simple `Issuer` (or even `iss` when talking about claims) in the details of implementation. 

So, _the_ issuer is actually an entity, but on the API level we're naming the URL as such.